configure

```diff
@@ -126,6 +126,18 @@ sub targetUsage {
 
 ## arg processing and usage
 
+my %aliases = (
+  '--doc_group' => '--doc-group',
+  '--wireshark_cmake' => '--wireshark-cmake',
+  '--wireshark_build' => '--wireshark-build',
+  '--wireshark_lib' => '--wireshark-lib',
+);
+for my $arg (@ARGV) {
+  if (exists $aliases{$arg}) {
```

Comment:
Doesn't look like this will work when an element of `@ARGV` is `--wireshark_build=/foo/bar`?

Reply:
Good point, I don't usually use that style so I didn't think of that case. A simple regex should cover that though.